shared/wallets/link-existing/index.js

```diff
@@ -94,6 +93,7 @@ class LinkWallet extends React.Component<Props, State> {
         */
         throw new Error('LinkExistingWallet: Unexpected value for `view` encountered: ' + this.state.view)
     }
+    // TODO: Refactor to use WalletPopup
```

Comment:
`LinkExisting` doesn't seem to have been worked for mobile at all, so it seemed best to save the changing `MaybePopup` to `WalletPopup` for when that happens.